lib/ansible/utils/unicode.py

```diff
@@ -215,7 +215,7 @@ def to_bytes(obj, encoding='utf-8', errors='replace', nonstring=None):
         return obj
     elif nonstring == 'simplerepr':
         try:
-            simple = binary_type(obj)
+            simple = str(obj)
         except UnicodeError:
```

Comment:
This one is wrong as using str() would return the text type on python3 and this function should only return byte type on both python versions.